main.go

```diff
@@ -70,6 +71,9 @@ func mainWithError() error {
 			serviceConfig.Name = name
 			serviceConfig.Source = source
 
+			serviceConfig.ResourceRetries = v.GetInt(f.Service.Resources.Retries)
```

Comment:
My configuration management doesn't feel just right yet. Will probably improve this further somewhere in the next few PRs, when we have some more arguments being passed around.